packages/miniflare/test/index.spec.ts

```diff
@@ -1087,3 +1087,31 @@ unixSerialTest(
 		t.is(await res.text(), "When I grow up, I want to be a big workerd!");
 	}
 );
+
+test("Miniflare: allows the use of unsafe eval bindings", async (t) => {
+	const log = new TestLog(t);
+
+	const mf = new Miniflare({
+		log,
+		modules: true,
+		script: `
+        export default {
+            fetch(req, env, ctx) {
+                const three = env.UNSAFE_EVAL.eval('2 + 1');
+
+                const fn = env.UNSAFE_EVAL.newFunction(
+                    'return \`the computed value is \${n}\`', '', 'n'
+                );
+
+                return new Response(fn(three));
+            }
+        }
+    `,
```

Comment:
mh.... all the other `script`s do use spaces... so having spaces here is more consistent with those 🤔 

<details>
<summary>examples</summary>

![Screenshot 2023-11-02 at 11 42 11](https://github.com/cloudflare/workers-sdk/assets/61631103/8c8367d4-a1d8-4ee0-a4f8-e4bc9f3b89f1)

![Screenshot 2023-11-02 at 11 42 25](https://github.com/cloudflare/workers-sdk/assets/61631103/eeb369d0-3478-4a46-88f5-e2534000b6cb)

![Screenshot 2023-11-02 at 11 42 40](https://github.com/cloudflare/workers-sdk/assets/61631103/0935ea41-c11e-459c-95f3-c379a68d21d2)

</details>

I don't mind using tabs here but if I do this would be the only script in the file indented with tabs, is that ok?

or shall I keep it as is and we can look later into just consistently applying tabs to all scripts?